packages/smooth_app/lib/pages/preferences/user_preferences_settings.dart

```diff
@@ -223,11 +225,59 @@ class _PrivacySettings extends StatelessWidget {
         const _CrashReportingSetting(),
         const UserPreferencesListItemDivider(),
         const _SendAnonymousDataSetting(),
+        const UserPreferencesListItemDivider(),
+        const _AdvancedSettings(),
       ],
     );
   }
 }
 
+class _AdvancedSettings extends StatelessWidget {
+  const _AdvancedSettings({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () async {
+        await AppSettings.openAppSettings();
+      },
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: <Widget>[
+          Expanded(
+            child: ListTile(
+              title: Text(
+                'Advanced App Settings',
```

Comment:
Could you add this String to lib/l10n/app_en.arb

Reply:
Yes I will add it.